pmm-app/tests/pages/dashboardPage.js

```diff
@@ -296,6 +296,13 @@ module.exports = {
       'MySQL Table Definition Cache',
     ],
   },
+
+  mySQLInstanceOverview: {
+    url: 'graph/d/mysql-instance-overview/mysql-instances-overview?orgId=1&from=now-2m&to=now&refresh=1m',
+    serviceName:
+      "//label[contains(text(), 'Service Name')]/following-sibling::value-select-dropdown/descendant::a[@class='variable-value-link']",
```

Comment:
This looks a bit too complicated according to the analyzer https://davertmik.github.io/locator